charts/datadog/templates/cluster-agent-deployment.yaml

```
@@ -241,6 +241,8 @@ spec:
           {{- if .Values.datadog.apm.instrumentation.enabled }}
           - name: DD_APM_INSTRUMENTATION_ENABLED
             value: {{ .Values.datadog.apm.instrumentation.enabled | quote }}
+          - name: DD_ADMISSION_CONTROLLER_AUTO_INSTRUMENTATION_INIT_RESOURCES_MEMORY
+            value: 100Mi
```

```suggestion
            value: 128Mi
```
This is set to `128Mi` in https://github.com/DataDog/helm-charts/pull/1520 and https://github.com/DataDog/datadog-operator/pull/1401

---

scratch that - found where 100mi comes from ^^.